zimui/src/components/TopicHome.vue

```diff
@@ -149,6 +150,11 @@ const parentSlug = (): string | null => {
         </div>
       </div>
     </div>
+    <ToolBar
+      v-if="topic.parents.length == 0"
+      :topics="getTopicSections(topic.sections)"
+      :key="slug"
```

Comment:
What is this `slug`?

Reply:
slug is unique for every topic ( using it as url ). So using it as a key here.

Reply:
But we are not in a `v-for`, so why do you need this `key` attribute?

Btw, I only realize this now but you should rename the Toolbar prop `sections` instead of `topics` to avoid confusion

Reply:
changed topics to sections. 
Removed key.

Why I kept keys?
I kept the option to display ToolBar on every topic page open , so i thought keys can be used for forced replacement of ToolBar component but I suppose that is not necessary because anyway we have watch(props, ...) inside ToolBar. Hence removing keys.

Question?
I am displaying toolbar only for those topics who has no parents, that is home page, just like endless ui. Though, I think having toolbar in every topic page is quite useful